src/client/automation/playback/click/index.js

```diff
@@ -150,16 +151,13 @@ export default class ClickAutomation {
         return cursor.leftButtonDown()
             .then(() => {
                 this._raiseTouchEvents();
-
-                var isBodyElement         = domUtils.isBodyElement(this.eventArgs.element);
-                var isContentEditable     = domUtils.isContentEditableElement(this.eventArgs.element);
-                var isContentEditableBody = isBodyElement && isContentEditable;
                 var activeElement         = domUtils.getActiveElement();
 
+                this.beforeMouseDownActiveElement = activeElement;
+
                 // NOTE: in IE focus is not raised if element was focused
                 // before click, even if focus is lost during mousedown
-                this.eventState.simulateFocus = !browserUtils.isIE || activeElement !== this.eventArgs.element ||
-                                                isContentEditableBody;
+                this.eventState.simulateFocus = !browserUtils.isIE;
```

Comment:
If now `this.eventState.simulateFocus` is assigned only in `focus` method for IE https://github.com/DevExpress/testcafe/pull/843/files#diff-0b8d22328e7dddcc102e65786aa60657R224
we can move this part of code there too